ledger-api/rs-grpc-akka/src/main/scala/com/digitalasset/grpc/adapter/AkkaExecutionSequencer.scala

```diff
@@ -67,7 +67,8 @@ private[grpc] class RunnableSequencingActor extends Actor with ActorLogging {
       try {
         runnable.run()
       } catch {
-        case NonFatal(t) => log.error("Unexpected exception while executing Runnable: {}", t)
+        case NonFatal(t) =>
+          log.error(s"Unexpected exception while executing Runnable ($runnable): {}", t)
```

Comment:
You can also do this to likely see the stacktrace
```suggestion
          log.error(t, s"Unexpected exception while executing Runnable ($runnable): {}", t)
```